roles/openshift_master/tasks/main.yml

```diff
@@ -201,7 +201,7 @@
     line: "{{ item.line }}"
   with_items:
     - regex: '^OPTIONS='
-      line: "OPTIONS=--loglevel={{ openshift.master.debug_level }} --listen=https://0.0.0.0:8443 --master=https://{{ openshift.common.ip }}:8443"
+      line: "OPTIONS=--loglevel={{ openshift.master.debug_level }} --listen=https://{{ openshift.master.bind_addr }}:{{ openshift.master.api_port }} --master=https://{{ openshift.common.ip }}:{{ openshift.master.api_port }}"
```

Comment:
Yeah, the default firewall rules are a known issue. iirc, we couldn't rely on the openshift_fact generated defaults to feed into os_firewall module based on the role dependencies.

I've mostly been dragging my feet on it until we can move to only supporting firewalld.